src/main/java/com/thevoxelbox/voxelsniper/brush/type/FlatOceanBrush.java

```diff
@@ -16,8 +16,14 @@
     private static final int DEFAULT_WATER_LEVEL = 29;
     private static final int DEFAULT_FLOOR_LEVEL = 8;
 
-    private int waterLevel = DEFAULT_WATER_LEVEL;
-    private int floorLevel = DEFAULT_FLOOR_LEVEL;
+    private int waterLevel;
+    private int floorLevel;
+
+    @Override
+    public void loadProperties() {
+        this.waterLevel = getIntegerProperty("default-water-level", DEFAULT_WATER_LEVEL);
```

Comment:
Values are not updated because brushes are stored in the toolkit, unbind it and assign it again weird stuff here, reload is just supposed to take account of new values _later_ imo.
No idea about the +1, that was always been the case.

Reply:
We can likely address the +1, from what I've seen it serves no purpose.

Reply:
Humm, just noticed and wondered, there is no "height" in FlatOceanBrush, wdym exactly, +1 only adds a layer of water on this brush. Are you somehow talking about another one?

Reply:
I'll get back to that later.
![](https://i.imgur.com/WMiotFa.jpg)
Is this how the brush is supposed to look like? yo 5 & yl 3
Looks alike the canyon brush just in weird? I imagined it would act like any other brush and putting down the blocks on top of the block you target on, instead of erasing a whole chunk.

Reply:
Oceans brushes currently recreate the whole terrain indeed, maybe to add to our to-do list?

Reply:
Likely, removing the entire chunk is eh.. weird to place a few rows of water and dirt. Can also add to the list to create an option to specify material instead of classic dirt.